Compression/src/CompressDigiMCsCheck_module.cc

```diff
@@ -21,6 +21,25 @@ using namespace std;
 namespace mu2e {
 
   class CompressDigiMCsCheck : public art::EDAnalyzer {
+
+    struct Config {
+      using Name=fhicl::Name;
+      using Comment=fhicl::Comment;
+
+      fhicl::Atom<art::InputTag> oldStrawDigiMCTag{Name("oldStrawDigiMCTag"), Comment("InputTag for uncompressed StrawDigiMCCollection")};
+      fhicl::Atom<art::InputTag> newStrawDigiMCTag{Name("newStrawDigiMCTag"), Comment("InputTag for compressed StrawDigiMCCollection")};
+      fhicl::Table<SimParticleTimeOffset::Config> oldTOff{Name("OldTimeOffsets"), Comment("TimeOffsets for uncompressed SimParticleCollection")};
+      fhicl::Table<SimParticleTimeOffset::Config> newTOff{Name("NewTimeOffsets"), Comment("TimeOffsets for compressed SimParticleCollection")};
+      fhicl::Atom<art::InputTag> strawDigiMCIndexMapTag{Name("strawDigiMCIndexMapTag"), Comment("If a StrawDigiMCIndexMap was passed to the compression originally, then pass it here too")};
+      fhicl::Atom<art::InputTag> oldCrvDigiMCTag{Name("oldCrvDigiMCTag"), Comment("InputTag for uncompressed CrvDigiMCCollection")};
+      fhicl::Atom<art::InputTag> newCrvDigiMCTag{Name("newCrvDigiMCTag"), Comment("InputTag for compressed CrvDigiMCCollection")};
+      fhicl::Atom<art::InputTag> crvDigiMCIndexMapTag{Name("crvDigiMCIndexMapTag"), Comment("If a CrvDigiMCIndexMap was passed to the compression originally, then pass it here too")};
+      fhicl::Atom<art::InputTag> oldCaloShowerSimTag{Name("oldCaloShowerSimTag"), Comment("InputTag for uncompressed CaloShowerSimCollection")};
```

Comment:
I don't think we will ever compress ShowerSim given the new workflows.  It doesn't hurt to keep it though.